Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/day/tabs/location/ContactDiaryLocationListFragment.kt

```diff
@@ -49,7 +48,11 @@ class ContactDiaryLocationListFragment : Fragment(R.layout.contact_diary_locatio
         viewModel.uiList.observe2(this) {
             locationListAdapter.update(it)
 
-            binding.contactDiaryLocationListNoItemsGroup.setInvisible(it.isNotEmpty())
+            if (it.isNotEmpty()) {
+                binding.contactDiaryLocationListNoItemsGroup.visibility = View.GONE
+            } else {
+                binding.contactDiaryLocationListNoItemsGroup.visibility = View.VISIBLE
+            }
```

Comment:
Since we have `core-ktx` already. can you please use `isGone`?
this will simplify the expression to:

`binding.contactDiaryLocationListNoItemsGroup.isGone = it.isNotEmpty()`

Reply:
Nice one, updated it for all